packages/viewer/src/lib/scenes/queries.ts

```diff
@@ -46,6 +46,14 @@ interface AllSelectedQueryExpression {
   type: 'all-selected';
 }
 
+interface AllVisibleQueryExpression {
+  type: 'all-visible';
+}
+
+interface AllHiddenQueryExpression {
```

Comment:
Do we have a known use-case for the hidden query?

I may recommend us not adding that until it's needed.